vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,16 +112,72 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
         exhibition.setStartSpace(startSpace);
         exhibition.setTitle(title);
         exhibition.setMode(exhibitMode);
-        exhibitManager.updateExhibitionLanguages(exhibition,languages,defaultLanguage);
-    
-        if(exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
+
+        exhibitManager.updateExhibitionLanguages(exhibition, languages, defaultLanguage);
+
+        if (exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
 
             exhibition.setCustomMessage(customMessage);
         }
+
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
```

Comment:
why does this need to be casted to `Exhibition` here?

Reply:
because exhibitManager.storeExhibition(exhibition) is returning IExhibition type

Reply:
yes, but why would it need to be `Exhibition` here?

Reply:
this still, why does exhibition need to be cast?